scripts/sql/233_mailmaster.up.sql

```diff
@@ -0,0 +1,96 @@
+INSERT INTO plugin_metadata (id,name,description,type,icon,deleted,created_on,created_by,updated_on,updated_by)
+VALUES (nextval('id_seq_plugin_metadata'),'Mail Master v1.0.0','The Plugin is designed for sending bulk emails through SMTP.','PRESET','https://raw.githubusercontent.com/devtron-labs/devtron/main/assets/MailMaster.png',false,'now()',1,'now()',1);
```

Comment:
Can we have a description like? Send bulk emails directly through your preferred SMTP server